tests/utils/test_files.py

```diff
@@ -64,3 +74,46 @@ def test_rename_file_original_does_not_exist(tmp_path: Path):
     # THEN a FileNotFoundError should be raised
     with pytest.raises(FileNotFoundError):
         rename_file(file_path=file_path, renamed_file_path=renamed_file_path)
+
+
+def test_get_creation_date(tmp_path_factory, timestamp_now: datetime):
+    """Test to retrieve the creation date of a directory."""
+
+    # GIVEN a directory that is created now
+    directory_path: Path = tmp_path_factory.mktemp("some_dir")
+
+    # WHEN retrieving the creation time stamp
+    creation_time_stamp: float = get_directory_creation_time_stamp(directory_path)
+    # THEN the creation time stamp is returned
```

Comment:
I think assertions regarding value should be added, maybe that the timestamp > yesterday, for example. There is a yesterday fixture somewhere

Reply:
its a tmp directory and I have not found a way to create a directory that is X amount of time old.